### ISSUE TITLE
Adapting your single-layer LSTM model for binary classification

### ISSUE THREAD

**Issue by ronikobrosly** [suriyadeepan/rnn-from-scratch#5]
Hello, thanks for making this code available. It's useful because I'm trying to create a custom, LSTM cell (with additional logic for the "forget" gate), and you've clearly coded out the internal logic of the cell. 

This isn't so much an issue on the repo I'm writing about. I'm wondering if you could explain and possibly show how to adapt the model for predicting a binary outcome. Let's say the new outcome variable is made like this:

```
from sklearn.preprocessing import OneHotEncoder, LabelBinarizer, LabelEncoder

_, Y = make_classification(n_samples = 118929, n_classes = 2, n_features=2, n_redundant=0, n_informative=1, n_clusters_per_class=1)
label_encoder = LabelEncoder()
integer_encoded = label_encoder.fit_transform(Y)
onehot_encoder = OneHotEncoder(sparse=False)
integer_encoded = integer_encoded.reshape(len(integer_encoded), 1)
onehot_encoded = onehot_encoder.fit_transform(integer_encoded)
Y = onehot_encoded
```

I feel very close but I'm getting errors here: tf.nn.sparse_softmax_cross_entropy_with_logits(logits=logits, labels=ys_)

Thank you!
